contracts/xasset/src/collateralized.rs

```diff
@@ -23,7 +23,7 @@ pub enum CDPStatus {
 #[loam_sdk::soroban_sdk::contracttype]
 #[derive(Clone)]
 /// Collateralized Debt Position for a specific account
-pub struct CDP {
+pub struct CDPContract {
```

Comment:
This name changes because `CDP` is defined twice within the xasset contract, and when I get to other processes with the orchestrator, this causes an error. Open to other name ideas.